core/trip.c

```diff
@@ -74,10 +74,12 @@ bool is_trip_before_after(const struct dive *dive, bool before)
 {
 	int idx = get_idx_by_uniq_id(dive->id);
 	if (before) {
-		if (idx > 0 && get_dive(idx - 1)->divetrip)
+		struct dive *d = get_dive(idx - 1);
```

Comment:
You could write this as `const struct dive *`, to make clear that die is not modified.

Reply:
good! will do